cypress/integration/dashboard/stats_spec.js

```diff
@@ -17,9 +17,12 @@ describe('Homepage stats', () => {
   it('should show a message pushing champions to use Zen when no stats is available', () => {
     cy.route('/api/2.0/users/instance', 'fx:parentLoggedIn').as('loggedIn');
     cy.route('POST', '/api/2.0/dojos/load-dojo-users', { response: [] }).as('dojoUsers');
+    cy.route('POST', '/api/2.0/events/applications/search', []).as('bookedChildren');
     cy.visit('/home');
     cy.wait('@loggedIn');
     cy.wait('@userDojos');
+    cy.wait('@bookedChildren');
+    cy.wait('@dojoUsers');
```

Comment:
Waiting for dojoUsers twice? Is this one meant to be userDojos like below?

Reply:
Technically, it's not wrong.
There are 2 calls to @dojoUsers : one for o13 and one for u13